example/gardener-local/kind-ha/cluster-skaffold.yaml

```diff
@@ -0,0 +1,176 @@
+        authorization-webhook-cache-unauthorized-ttl: "0"
+      extraVolumes:
+      - name: gardener
+        hostPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        mountPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        readOnly: true
+        pathType: File
+  - |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
+    maxPods: 500
+    serializeImagePulls: false
+    registryPullQPS: 10
+    registryBurst: 20
+
+- role: worker
+  labels:
+    topology.kubernetes.io/zone: "2"
+  image: kindest/node:v1.24.0
+  extraMounts:
+  - hostPath: example/gardener-local/controlplane
+    containerPath: /etc/gardener/controlplane
+    readOnly: true
+  - hostPath: dev/local-backupbuckets
+    containerPath: /etc/gardener/local-backupbuckets
+  - hostPath: dev/local-registry
+    containerPath: /etc/gardener/local-registry
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    apiServer:
+      extraArgs:
+        authorization-mode: RBAC,Node,Webhook
+        authorization-webhook-config-file: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        authorization-webhook-cache-authorized-ttl: "0"
+        authorization-webhook-cache-unauthorized-ttl: "0"
+      extraVolumes:
+      - name: gardener
+        hostPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        mountPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        readOnly: true
+        pathType: File
+  - |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
+    maxPods: 500
+    serializeImagePulls: false
+    registryPullQPS: 10
+    registryBurst: 20
+
+- role: worker
+  labels:
+    topology.kubernetes.io/zone: "3"
+  image: kindest/node:v1.24.0
+  extraMounts:
+  - hostPath: example/gardener-local/controlplane
+    containerPath: /etc/gardener/controlplane
+    readOnly: true
+  - hostPath: dev/local-backupbuckets
+    containerPath: /etc/gardener/local-backupbuckets
+  - hostPath: dev/local-registry
+    containerPath: /etc/gardener/local-registry
+  kubeadmConfigPatches:
+  - |
+    kind: ClusterConfiguration
+    apiServer:
+      extraArgs:
+        authorization-mode: RBAC,Node,Webhook
+        authorization-webhook-config-file: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        authorization-webhook-cache-authorized-ttl: "0"
+        authorization-webhook-cache-unauthorized-ttl: "0"
+      extraVolumes:
+      - name: gardener
+        hostPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        mountPath: /etc/gardener/controlplane/auth-webhook-kubeconfig-skaffold.yaml
+        readOnly: true
+        pathType: File
+  - |
+    apiVersion: kubelet.config.k8s.io/v1beta1
+    kind: KubeletConfiguration
+    maxPods: 500
+    serializeImagePulls: false
+    registryPullQPS: 10
+    registryBurst: 20
```

Comment:
Same question here. Also, why does the control-plane node have zone "0"? Do we need this?
Also, why does the control-plane node in `cluster-local.yaml` not have this label?